### PR TITLE
Added Meta to app

### PR DESCRIPTION
**Pull request by eliottha**
Test with some live_version strings.

TODO:
Add app.detail data (eg. categories, ...)
Update app.live_version data (go to app.edit_version if data is null)
app.edit_data if values are null/empty
...

I'm not finished with this! I will finish and upload the final result tomorrow!

### CODE REVIEW

File: server.rb [hunk 1]
```diff
@@ -56,7 +99,9 @@
 	if store_front == nil 
 		store_front = ""
 	end
-	Spaceship::Tunes.login(username, password)
+	client = Spaceship::Tunes.login(username, password)
+    client.team_id = request.env['HTTP_TEAM_ID']
```
Comment (RishabhTayal) [inline server.rb]: Do we need to set the `team_id`?

File: server.rb [hunk 2]
```diff
@@ -46,6 +45,50 @@
...
+#secondsubseccat = app.details.secondary_second_sub_category
+
+# Gets screenshots of live_version app
+get '/app/metadata' do
+	content_type :json
+	username = request.env['HTTP_USERNAME']
+	password = request.env['HTTP_PASSWORD']
+	bundle_id = params[:bundle_id]
+	client = Spaceship::Tunes.login(username, password)
+    client.team_id = request.env['HTTP_TEAM_ID']
```
Comment (RishabhTayal) [inline server.rb]: Let's read all the header in one place. Move this line after `password = request.env['HTTP_PASSWORD']`

File: server.rb [hunk 3]
```diff
@@ -46,6 +45,50 @@
...
+
+	version =  Spaceship::Tunes::Application.find(bundle_id)
+	p version
```
Comment (RishabhTayal) [inline server.rb]: Remove print statements.

**Review by RishabhTayal — CHANGES_REQUESTED** [review]

File: server.rb (hunk 1 above)
Comment (eliottha) [reply server.rb]: Removed

**Comment by RishabhTayal** [comment]
@Eliottiosdevs Please resolve merge conflicts and it's ready to merge.

**Comment by eliottha** [comment]
Done btw

File: server.rb [hunk 4]
```diff
@@ -143,7 +174,6 @@
 end
 
 # Get List of testers
-get '/testers' do
```
Comment (RishabhTayal) [inline server.rb]: Why was this kine removed?

**Review by RishabhTayal — CHANGES_REQUESTED** [review]

File: server.rb (hunk 4 above)
Comment (eliottha) [reply server.rb]: Removed by abuse when I fixed merge conflict, thanks for verifying!

**Review by RishabhTayal — APPROVED** [review]

**Comment by RishabhTayal** [comment]
Merged!

**Comment by eliottha** [comment]
Thanks @RishabhTayal

**Pull request merged**